app/views/componentsP5/table_3/row_1.scala.xml

```diff
@@ -0,0 +1,196 @@
+@import views.xml.componentsP5.table_1.consignee
+        @import views.xml.componentsP5.table_1.total_gross_mass
+
+        @(ie029Data: models.P5.departure.IE029Data)
+
+<fo:table-row height="16%">
+
+    <fo:table-cell number-columns-spanned="2">
+        <fo:block>
+            <fo:table width="100%" table-layout="fixed">
+                <fo:table-body>
+                    <fo:table-row height="35.6%" border-bottom-style="solid" border-bottom-width="1pt"
+                                  border-bottom-color="black"> <!-- todo where is this extra .25% -->
+                        <fo:table-cell display-align="top" margin-left="2%" padding-bottom="163%" padding-top="5%">
```

Comment:
This seems like a huge padding bottom, is this correct?

Reply:
removed the bottom border and added it to top of next row hence getting rid of the 163%